backend/app/modules/user/antisybil/service/initial.py

```diff
@@ -76,6 +85,26 @@ def _verify_address_is_not_delegated(self, user_address: str):
             raise AddressAlreadyDelegated()
 
 
+guest_list_stamp_providers = [
```

Comment:
it can be uppercased and in constants.py too and

Reply:
+1